server.js

```diff
@@ -21,15 +22,16 @@ app.use(express.json());
 app.use(express.json({ type: 'application/activity+json' }));
 app.use(session());
 
-app.use((req, res, next) => {
+app.use(async (req, res, next) => {
   res.locals.loggedIn = req.session.loggedIn;
+  const { displayName } = await getActorInfo();
```

Comment:
Feels not great to make this database call before every page load but I think this is just the world we live in